Build fail on PR88

./src/helpers/userDataFormatter.ts:18:9
Type error: Property 'getsMail' is missing in type '{ onbc_id: any; name: string; email: any; avatar: any; bio: string; patreon: { state: string; }; favourites: undefined[]; reads: undefined[]; wishlist: undefined[]; ratings: undefined[]; readonly isPatron: boolean; }' but required in type 'DisplayUser'.
  16 |   };
  17 | 
> 18 |   const formattedUser: DisplayUser = {
     |         ^
  19 |     onbc_id,
  20 |     name: '',
  21 |     email,
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! book-people@0.1.0 build: `next build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the book-people@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:

--- a/src/helpers/userDataFormatter.ts
+++ b/src/helpers/userDataFormatter.ts
@@ -24,6 +24,7 @@ const userDataFormatter = (user): DisplayUser => {
     patreon: {
       state: patreonStateChecker(patreon),
     },
+    getsMail: true,
     favourites: [],
     reads: [],
     wishlist: [],

